### PR TITLE
Update/chapter 4

### PR DESCRIPTION
**Pull request by hyckomatej**

### CODE REVIEW

**Review by lukacan — CHANGES_REQUESTED** [review]

File: docs/handbook/chapter4/token-2022.md [hunk 1]
```diff
@@ -1,32 +1,56 @@
...
+!!! note
 
-Current mint extensions include 14 extensions. Some of the most important are:
+    - **Mint extensions** are added on top of the original Solana Token Program and extend the capabilities of tokens.
+    - **Account extensions** are added on top of Solana accounts and add account-related features.
 
-- **Confidential transfers**: Allow confidential transfers between participating users without revealing the amount of the transfer.
-- **Transfer fees**: Allow transfer fees to be charged on each transfer and sent to a defined account.
-- **Mint close authority**: Allows owners to close mint accounts and reclaim the lamports on the mint account.
+Some of the mint extensions:
```
[the remark this answers is not in the document]
Comment (hyckomatej) [reply docs/handbook/chapter4/token-2022.md]: some of the extensions inside paper are not in the documentation https://spl.solana.com/token-2022.

[the remark this answers is not in the document]
Comment (hyckomatej) [reply docs/handbook/chapter4/token-2022.md]: but they are [here](https://github.com/solana-labs/solana-program-library/blob/master/token/program-2022/src/extension/mod.rs). should i add them as well?

**Review by lukacan — CHANGES_REQUESTED** [review]
private communication

**Comment by lukacan** [comment]
@hyckomatej I made some small updates can you please check ?

**Review by lukacan — APPROVED** [review]

**Pull request merged**